### DISCUSSION TITLE
Select All checkbox on dataTable with reflow

### DISCUSSION THREAD

**Discussion by AndreMenezesDev** [primefaces/community#1513] [PrimeFaces]
I have a dataTable with checkbox selection. It works well on desktop. But on mobile screen, with reflow attribute, checkbox for each row is fine, but the "Select All" checkbox in the header is missing. Seeing how headers are stacked on reflow mode, I am not sure how it's supposed to work. Can't find an example in the demo. I am not sure if it's a bug, or my layout/stylesheet somehow broke it. Could anyone confirm if "Select All" checkbox is supported with reflow?

```
<p:dataTable value="#{bean.list}" var="row" rowIndexVar="rowIndex" selection="#{bean.selectedRows}" rowKey="#{row.id}" sortBy="#{row.branchName}" sortMode="single" disabledSelection="#{row.selectionDisabledTrue}" rowSelectMode="checkbox" reflow="true">
	<p:headerRow>
		<p:column colspan="8">
			<t:div style="padding: 6px 4px">
				<h:outputText value="#{row.branchName}" />
			</t:div>
		</p:column>
	</p:headerRow>

	<p:column id="selectCol" selectionMode="multiple" />
	...
</p:dataTable>
```

**Comment by melloware** [comment]
That is correct on Reflow the header is redrawn and does not have a select all checkbox
